### PR TITLE
README updates for iOS Simulator and Android Emulator setup

### PR DESCRIPTION
**Pull request by samgutentag**
for real this time, gross merge but all the info made it 🙃

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,11 +20,18 @@ In order to interact with any package contained in this repository you will need
 
 - [**NVM**](https://github.com/nvm-sh/nvm/blob/master/README.md)
 - [**Homebrew**](https://brew.sh/)
-- [**Node.js@18.x.x**](https://nodejs.org/)
+- [**Node.js@16.x.x**](https://nodejs.org/) `16.x.x`
```
Comment (tyler-whitman) [inline README.md:23]: This should now be node 18

File: README.md [hunk 2]
```diff
@@ -20,11 +20,18 @@ In order to interact with any package contained in this repository you will need
...
 - [**Cocoapods**](https://cocoapods.org/)
-- [**Xcode@14.2**](https://xcodereleases.com/)
-- [**Android Studio**](https://developer.android.com/studio)
+- [**Xcode**](https://developer.apple.com/xcode/) `14.2`
+  - [Download older versions of Xcode here](https://developer.apple.com/download/all/?q=xcode%2014.2)
+- [**Android Studio**](https://developer.android.com/studio) `Chipmunk`
+  - [Download older versions of Android Studio here](https://developer.android.com/studio/archive)
+  - [Install a Virtual Device](https://developer.android.com/studio/run/emulator#avd) (`Nexus 5` with `UpsideDownCake` good default)
+  - When [installing the Android SDK](https://reactnative.dev/docs/environment-setup#android-sdk) use these:
+  - `Android 12.0`
```
Comment (tyler-whitman) [inline README.md:32]: We can bump this to 13 now too if we want. 12 should still work fine though if you want to leave it. 12 is probably more common out there anyway

**Review by tyler-whitman — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (samgutentag) [reply README.md:32]: I can note that, Andoird 12 was the latest supported by the Chipmunk version of Android Studio that I could find

File: README.md (hunk 1 above)
Comment (samgutentag) [reply README.md:23]: ah nice catch, just reran everything with node 18, all clear 👍

File: README.md [hunk 3]
```diff
@@ -20,11 +20,18 @@ In order to interact with any package contained in this repository you will need
...
-- [**Xcode@14.2**](https://xcodereleases.com/)
-- [**Android Studio**](https://developer.android.com/studio)
+- [**Xcode**](https://developer.apple.com/xcode/) `14.2`
+  - [Download older versions of Xcode here](https://developer.apple.com/download/all/?q=xcode%2014.2)
+- [**Android Studio**](https://developer.android.com/studio) `Chipmunk`
+  - [Download older versions of Android Studio here](https://developer.android.com/studio/archive)
+  - [Install a Virtual Device](https://developer.android.com/studio/run/emulator#avd) (`Nexus 5` with `UpsideDownCake` good default)
+  - When [installing the Android SDK](https://reactnative.dev/docs/environment-setup#android-sdk) use these:
+  - `Android 13.0`
+    - `Android SDK Platform 31`
```
Comment (tyler-whitman) [inline README.md:33]: I think for 13, this should be 33

**Review by tyler-whitman — APPROVED** [review]
Looks good, thanks for the fixes just one other minor thing

**Comment by Perronef5** [comment]
@samgutentag Another merge conflict 🙃

**Comment by ChewingGlass** [comment]
@samgutentag want to resolve the comments here?

**Pull request closed without merging**